content/credits.md

```diff
@@ -5,16 +5,12 @@ disableToc: true
 
 ## Contributors
 
-{{< raw_html >}}
+Thanks to them <i class="fas fa-heart"></i> for making Open Source Software a better place ! <br/>
 
-Thanks to them <i class="fas fa-heart"></i> for making Open Source Software a better place ! <br>
-
-<br>
+<br/>
 
 {{% ghcontributors "https://api.github.com/repos/kubeshop/botkube/contributors?per_page=100" %}}
```

Comment:
It doesn't work. Personally, not sure if needed. What will be your approach? Use React component?

Reply:
I left it for later :P yes but I will use React component for that.